recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
 
   run:
     - python
-    - botocore ==1.5.83
+    - botocore ==1.8.23
     - colorama >=0.2.5,<=0.3.7
     - docutils >=0.10
     - rsa >=3.1.2,<=3.5.0
```

Comment:
TODO: fix the next line to match https://github.com/aws/aws-cli/blob/1.14.19/setup.py#L30 once https://github.com/conda-forge/s3transfer-feedstock/pull/11 is merged